website/content/en/docs/advanced-topics/multi-arch.md

```diff
@@ -36,23 +43,169 @@ docker buildx build --push --platform linux/amd64,linux/arm64,linux/ppc64le,linu
+**Caveat**: When mirroring registries for disconnected installations (environments without internet connection) all the images referenced by a manifest list need to be copied, including images for architectures that may not be used in the environment.
 
-#### Operator Lifecycle Manager
+### Operator Lifecycle Manager
 
 For operators distributed through the [Operator Lifecycle Manager (OLM)][olm]:
 
-* [Bundle images][bundle] are not architecture-specific. They contain only plaintext kubernetes manifests and operator metadata.
+* [Bundle images][bundle] are not architecture-specific. They contain only plaintext Kubernetes manifests and operator metadata.
 * All image references in the ClusterServiceVersion should be manifest lists containing the pointers to the image manifests for the supported architectures.
 * Labels for OS and architectures can be set in the CSV. Please refer to the [Operator Lifecycle Management Documentation][olm_multiarch] for details.
 
-[manifest_list]: https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list
-[image_index]: https://github.com/opencontainers/image-spec/blob/main/image-index.md
+## Supporting Clusters with Multi-Architecture Compute Nodes
+The [Fundamentals](#Fundamentals) above aim to guide authors on the key steps to building and distributing operators that can run on multiple architectures. These instructions are sufficient when your cluster's compute nodes share the same architecture. However, operator authors should also understand the implications of running their operators in a cluster with multi-architecture compute nodes since it is not always guaranteed that the architectures of the compute nodes will match the architectures supported by the operator.
+
+### Safe Scheduling Using Node Affinity
+Node affinity is a mechanism exposed in a Kubernetes pod template that allows a `PodSpec` author to instruct the scheduler to restrict a pod to run only on (or with a preference for) nodes that meet specific criteria. To ensure that pods are always scheduled to nodes of compatible architecture, it is a best practice for authors to set node affinity requirements to ensure their operators and operands will only schedule to the nodes with architectures available to the images in the pod. If you don't do this, a container scheduled to an incompatible node will immediately crash with an `exec format error`, which will ultimately lead to an `ImagePullBackoff` event as the pod is restarted only to crash again with the same error.
+
+#### Determining the Architectures Supported by an Image
+For a given container image, you can check which architectures are supported by listing them by inspecting the manifest. Piping the output to the python `json.tool` module enables pretty-printed JSON output.
+```shell
+$ skopeo inspect --raw <image> | python -m 'json.tool'
```

Comment:
Might also be nice to show how this inspect can be done with `docker inspect ...`

Reply:
Added this back into this draft after showing how to do it with skopeo. Since the output is basically the same, I just showed the commands.